### PR TITLE
[Accessibility] Fixing quit test label

### PR DESCRIPTION
**Pull request by fnormand01**
# Description

This PR is simply fixing the "Quit test" button label. According to David, when selecting this button, the screen reader should say "Quit test" and not the description of the action itself like it does now.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Screenshot

(N/A)

# Testing

1.  Launch NVDA.
2.  Start sample test.
2.  Navigate using the keyboard and focus on "Quit test" button and make sure that the screen reader says "Quit test button".

# Checklist

- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [x] My changes generate no new compiler warnings
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [x] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)
- [x] My changes look good on IE 11+ and Chrome

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+services:
```
Comment (MichaelCherry) [inline .gitlab-ci.yml]: This seems unrelated to the rest of the PR. I feel like this should be in a separate PR

Comment (fnormand01) [reply .gitlab-ci.yml]: ohhh! I didn't want to commit that file.
Removed in commit #https://github.com/code-for-canada/project-thundercat/pull/490/commits/e6593b88a83a5aa4e03a24b1ece01c77e89d429a.

**Comment by fnormand01** [comment]
image
